docs/_documentations/troubleshooting.md

```diff
@@ -941,12 +941,16 @@ Check the Codewind output stream first when troubleshooting because it is partic
 
 If you report an issue, you will be asked to upload your logs.
 
-1. In VS Code, open **Help** > **Toggle Developer Tools**.
-2. Go to the **Console** tab.
-3. Enter *codewind.log* in the **Filter** box:
-<br>![Log location](images/cdt-vsc/logs-location.png)
+1. In VS Code, open the **Output** view.
+2. Select the Codewind output stream.
+3. The first line of the Codewind output stream is the location of the Codewind extension logs.
+<br><br>
+![Codewind output stream with log location](images/cdt-vsc/output_with_logpath.png)
+<br>
 4. Upload the contents of the log file with your issue report.
 
+Also see [Collecting Log Files and Environment Data](#collecting-log-files-and-environment-data). In VS Code this is accessible through the `Codewind: Capture Diagnostics` command.
```

Comment:
- Please use lower case for the section heading: "Collecting log files and environment data".
- Also, I've created a PR to include the `Codewind: Capture Diagnostics` command in the "Collecting log files and environment data" section along with the corresponding info for Eclipse. Maybe it's best to remove the sentence, "In VS Code this is accessible through the `Codewind: Capture Diagnostics` command." since the info is covered in the other section?